ewapt.py

```diff
@@ -469,6 +469,18 @@ def getPriceBase(cmd):
 
 	furns = ewitem.inventory(id_user= lookObject+ewcfg.compartment_id_decorate, id_server= playermodel.id_server, item_type_filter=ewcfg.it_furniture)
 
+	furniture_name_list = []
+	for furnName in furns:
+		furniture_name_list.append(furnName.get('name'))
+
+	if "LGBTQ+ desk" in furniture_name_list and "LGBTQ+ chair" in furniture_name_list and "LGBTQ+ couch" in furniture_name_list and "LGBTQ+ lamp" in furniture_name_list and "LGBTQ+ bed" in furniture_name_list:
```

Comment:
these will be a pain in the ass to update and maintain. we should keep these as lists in ewcfg, ideally populate the lists automatically via a "set" attribute in EwFurniture